GitUp/Application/AppDelegate.m

```diff
@@ -613,6 +620,10 @@ - (IBAction)cloneRepository:(id)sender {
   }
 }
 
+- (IBAction)cloneRepository:(id)sender {
+  [self _cloneRepositoryFromURLString:@""];
```

Comment:
Why would that not work? It's the equivalent of what was there before, substituting in @"" for the urlString param passing in @"".